Backend/dev/test-tool/context-api/server.py

```
@@ -457,6 +471,29 @@ def _handle(self, method):
             self._send_json({"logs": logs})
             return True
 
+        if method == "POST" and path == "/api/redis/flushall":
+            try:
+                import subprocess
+                result = subprocess.run(
+                    ["redis-cli", "--cluster", "call", "localhost:30001", "flushall"],
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)